ui/components/multichain/connected-site-popover/connected-site-popover.stories.tsx

```diff
@@ -16,13 +18,11 @@ export default {
   },
 };
 
-const Template = (args) => {
-  return <ConnectedSitePopover {...args} />;
-};
 
-export const DefaultStory = Template.bind({});
+export const DefaultStory = {};
 
-export const ConnectedStory = Template.bind({});
-ConnectedStory.args = {
-  isConnected: true,
+export const ConnectedStory= {
+  args: {
+    isConnected: true,
+  }
```

Comment:
Updating stories here to latest [CSF](https://storybook.js.org/docs/api/csf) format